lamindb/_record.py

```diff
@@ -263,6 +279,48 @@ def _format_django_validation_error(record: Record, e: DjangoValidationError):
         return message
 
 
+def _get_model_params(record_class) -> list[tuple[str, str]]:
+    """Gets the parameters of a Record from the overloaded signature.
+
+    Example:
+        >>> get_record_params(bt.Organism)
+        >>> [('name', 'str'), ('taxon_id', 'str | None'), ('scientific_name', 'str | None')]
+    """
+    source = inspect.getsource(record_class)
+
+    # Find first overload that's not *db_args
+    pattern = r"@overload\s+def __init__\s*\(([\s\S]*?)\):\s*\.{3}"
+    overloads = re.finditer(pattern, source)
+
+    for overload in overloads:
+        params_block = overload.group(1)
+        if "*db_args" in params_block:
+            continue
```

Comment:
That's weird that this is never called in tests. Is there something wrong here?

Reply:
No we just want to ensure that we don't accidentally look at the signature that has `*db_args` in the args. I would have to find a Record class where this overload signature comes first. We currently don't have that but this makes the code safer if someone accidentally does that.

Reply:
Ok, understood! Can you please add this is as a comment in the code?